0029-stream/0029-stream.md

```diff
@@ -436,6 +444,24 @@ In other words, if a sender resends data (e.g. because a packet was lost), it MU
 
 Asset details exposed by this frame MUST NOT change during the lifetime of a Connection.
 
+#### 5.3.15. `StreamReceipt` Frame
+
+| Field | Type | Description |
+|---|---|---|
+| Stream ID | VarUInt | Identifier of the stream this frame refers to. |
+| Receipt | 65-Byte OctetString | Proof provided by the receiver of the total amount received on this stream |
```

Comment:
Why mandate the exact size of the receipt instead of leaving it open-ended for possible extension?

Reply:
:+1:
Should we also consider including a receipt version number?
Or could the receipt size be used to check compatibility, such as with receipt support discovery (https://github.com/interledger/rfcs/pull/570#discussion_r395327765)?

Reply:
Perhaps the receipt itself could have a version byte? Size might not be good enough if a future version is variable length. And if the receipt contains its own version info then STREAM doesn't need to emit extra information about receipt version

Reply:
changed receipt to `VarOctetString` and added version number: https://github.com/interledger/rfcs/pull/570/commits/1633e86180f268e4808f9011610f91b6f325096b